dotnet3.5/Polly/SynthesizeSpeechExample/SynthesizeSpeech/SynthesizeSpeech.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier:  Apache - 2.0
+
+using Amazon;
+using Amazon.Polly;
+using Amazon.Polly.Model;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace SynthesizeSpeech
+{
+    class SynthesizeSpeech
+    {
+        // The following example text to speech and saves it to an .mp3 file
+        // using Amazon Polly. It was created using the AWS SDK for .NET 3.5
+        // and .NET 5.0.
+
+        // Specify your AWS Region (an example Region is shown).
+        private static readonly RegionEndpoint serviceRegion = RegionEndpoint.USEast2;
+        private static IAmazonPolly polyClient;
+
+        private const string outputFileName = "speech.mp3";
+        private const string text = "Twas brillig, and the slithy toves did gyre and gimbol in the wabe";
+
+        static async Task Main()
+        {
+            polyClient = new AmazonPollyClient(serviceRegion);
+            var response = await PollySynthesizeSpeech(polyClient, text);
+            
+            if (response.ContentLength > 0) {
+                WriteSpeechToStream(response.AudioStream, outputFileName);
+            }
+        }
+
+        /// <summary>
+        /// Calls Amazon Polly SynthesizeSpeechAsync method to convert text
+        /// to speech.
+        /// </summary>
+        /// <param name="client">The Amazon Polly client object used to connect
+        /// to the Amazon Polly service.</param>
+        /// <param name="text">The text which will be converted to speech.</param>
+        /// <returns>A SynthesizeSpeechResponse object that includes an AudioStream
+        /// object with the converted text.</returns>
+        private static async Task<SynthesizeSpeechResponse> PollySynthesizeSpeech (IAmazonPolly client, string text)
+        {
+            var synthesizeSpeechRequest = new SynthesizeSpeechRequest()
+            {
+                OutputFormat = OutputFormat.Mp3,
+                VoiceId = VoiceId.Joanna,
+                Text = text
+            };
+
+            var synthesizeSpeechResponse =
+                await client.SynthesizeSpeechAsync(synthesizeSpeechRequest);
+            
+            return synthesizeSpeechResponse;
+        }
+
+        /// <summary>
+        /// Writes the AudioStream returned from the call to
+        /// SynthesizeSpeechAsync to a file in MP3 format.
```

Comment:
"MP3" here, but ".mp3" earlier?

Reply:
Yes. Here I'm talking about MP3 format and earlier had mentioned the extension .mp3. So, I've changed it to use MP3 (the format, not the extension) in both places.